### PR TITLE
Update audit-log-retention-policies.md

### LINKED ISSUE #12206: License Hyperlink to the 10 year Add-On license.
[Enter feedback here]
https://learn.microsoft.com/en-us/microsoft-365/compliance/audit-log-retention-policies?view=o365-worldwide

Please do add the hyperlink to buy the 10 year Add-On license.

Thank You.
auditlogretentiondashboardpriority
ediscovery-create-new-audit-retention-policy

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: e52e1877-191e-5dbe-b805-e755c736b985
* Version Independent ID: 69df5ed6-a86f-06d7-1156-e1de211f8e83
* Content: [Manage audit log retention policies - Microsoft Purview (compliance)](https://learn.microsoft.com/en-us/microsoft-365/compliance/audit-log-retention-policies?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/audit-log-retention-policies.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/audit-log-retention-policies.md)
* Service: **o365-seccomp**
* GitHub Login: @robmazz
* Microsoft Alias: **robmazz**

### PR DESCRIPTION
**Pull request by msbemba**
Add information on how to purchase the add-on
Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12206

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/audit-log-retention-policies.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ Audit (Premium) in Microsoft 365 provides a default audit log retention policy f
...
 - You can have a maximum of 50 audit log retention policies in your organization.
-- To retain an audit log for longer than 90 days (and up to 1 year), the user who generates the audit log (by performing an audited activity) must be assigned an Office 365 E5 or Microsoft 365 E5 license or have a Microsoft 365 E5 Compliance or E5 eDiscovery and Audit add-on license. To retain audit logs for 10 years, the user who generates the audit log must also be assigned a 10-year audit log retention add-on license in addition to an E5 license.
+- To retain an audit log for longer than 90 days (and up to 1 year), the user who generates the audit log (by performing an audited activity) must be assigned an Office 365 E5 or Microsoft 365 E5 license or have a Microsoft 365 E5 Compliance or E5 eDiscovery and Audit add-on license. To retain audit logs for 10 years, the user who generates the audit log must also be assigned a 10-year audit log retention add-on license in addition to an E5 license. For more information on how to purchase an Add-on, See [Buy an add-on](/microsoft-365/commerce/buy-or-edit-an-add-on?view=o365-worldwide#buy-an-add-on).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/audit-log-retention-policies.md]:
```suggestion
- To retain an audit log for longer than 90 days (and up to 1 year), the user who generates the audit log (by performing an audited activity) must be assigned an Office 365 E5 or Microsoft 365 E5 license or have a Microsoft 365 E5 Compliance or E5 eDiscovery and Audit add-on license. To retain audit logs for 10 years, the user who generates the audit log must also be assigned a 10-year audit log retention add-on license in addition to an E5 license. For more information on how to purchase an add-on, See [Buy an add-on](/microsoft-365/commerce/buy-or-edit-an-add-on#buy-an-add-on).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

File: microsoft-365/compliance/audit-log-retention-policies.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Audit (Premium) in Microsoft 365 provides a default audit log retention policy f
...
 - You can have a maximum of 50 audit log retention policies in your organization.
-- To retain an audit log for longer than 90 days (and up to 1 year), the user who generates the audit log (by performing an audited activity) must be assigned an Office 365 E5 or Microsoft 365 E5 license or have a Microsoft 365 E5 Compliance or E5 eDiscovery and Audit add-on license. To retain audit logs for 10 years, the user who generates the audit log must also be assigned a 10-year audit log retention add-on license in addition to an E5 license.
+- To retain an audit log for longer than 90 days (and up to 1 year), the user who generates the audit log (by performing an audited activity) must be assigned an Office 365 E5 or Microsoft 365 E5 license or have a Microsoft 365 E5 Compliance or E5 eDiscovery and Audit add-on license. To retain audit logs for 10 years, the user who generates the audit log must also be assigned a 10-year audit log retention add-on license in addition to an E5 license. For more information on how to purchase an add-on, See [Buy an add-on](/microsoft-365/commerce/buy-or-edit-an-add-on#buy-an-add-on).
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/audit-log-retention-policies.md]:
```suggestion
- To retain an audit log for longer than 90 days (and up to 1 year), the user who generates the audit log (by performing an audited activity) must be assigned an Office 365 E5 or Microsoft 365 E5 license or have a Microsoft 365 E5 Compliance or E5 eDiscovery and Audit add-on license. To retain audit logs for 10 years, the user who generates the audit log must also be assigned a 10-year audit log retention add-on license in addition to an E5 license. For more information about how to purchase an add-on, see [Buy an add-on](/microsoft-365/commerce/buy-or-edit-an-add-on#buy-an-add-on).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @msbemba, one more, please.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
Closing as per the comment in #12206

**Pull request closed without merging**